release.sh

```diff
@@ -204,14 +202,27 @@ else
     git commit -n -m "v$VERSION"
     git diff --stat --exit-code # check for uncommitted changes
     git tag -a "v$VERSION" -m "v$VERSION"
-    git push origin "v$VERSION"
+    git push "$ORIGIN" "v$VERSION"
 
     # Update npm (cdnjs update automatically)
-    yarn publish --new-version "${VERSION}"
+    # Fallback to npm publish, if yarn cannot authenticate, e.g., 2FA
+    yarn publish --new-version "$VERSION" || npm publish
 
     # Publish the website
+    # If gh-pages branch is protected, push to another branch
     pushd website
-    USE_SSH=true yarn publish-gh-pages
+    PUBLISH_GH_PAGES=$(USE_SSH=true yarn publish-gh-pages 2>&1 | tee /dev/tty || true)
+    if echo "$PUBLISH_GH_PAGES" | grep -qEi 'GH006|protected branch'; then
+        pushd build/KaTeX-gh-pages
+        git checkout -B "v$VERSION-gh-pages"
+        git push -u origin "v$VERSION-gh-pages"
```

Comment:
Should `origin` be `"$ORIGIN"`? Ah, I guess not -- this is meant for the forked repo.